admin-dev/themes/new-theme/scss/components/layout/_nav_bar.scss

```diff
@@ -213,6 +213,10 @@
         border-left: .25rem solid #25b9d7;
         padding-left: .638rem;
 
+        @at-root .sidebar-closed .link-levelone .link > span {
```

Comment:
Why `at-root`? You could just put it at root

Reply:
Seems clearer to me since it's under initial properties of the element

Reply:
Tihs is not the intended way of using at-root (https://sass-lang.com/documentation/at-rules/at-root)

Using it this way will make finding the scss declaration significantly harder to do when analyzing the final CSS styles, as it won't be placed in the expected part of the scss tree (the root).